### DISCUSSION TITLE
Welcome to axolotl Discussions!

### DISCUSSION THREAD

**Discussion by nanu-c** [axolotl-chat/axolotl#284] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MoemUT** [comment]
Okay then! This is such a friendly invitation and here it sits unheard and unanswered. I might as well reply.

Hi! 👋 I'm Moem. I'm a UT user and very much not a coder. I'm more of a creative maker who is also active in hackerspaces. But I'm here to support the development of Axolotl and if I can help in any way, I would be happy to.

**Comment by kmsgli** [comment]
Hey so I am having good success with axolotl and since the pinephone can now do MMS I have made the switch to pinephone as my daily driver!

Since axolotl has no push notifications I made a simple script that runs axolotl and a systemd timer that triggers the axolotl script every 15 minutes. It works great however I assume there is a way to just pull messages every 15 min than just run the command "axolotl". Interestingly enough when run from the command line it checks for messages tries to start the program fails but the message notifications actually come through. So it does work as a notification system but fails to open the app, however when i tap the icon the program does open as it should. I am wondering if maybe someone can help me polish this process a bit for others to get some use from it?

PS major props for your work nanu-c you are making the Linux life that much easier for all of us.

Thanks.

**Reply by Blackoverflow** [reply]
On Ubuntu Touch (UT), it is possible to receive notifications as long as axolotl is running (even in background and with locked screen):
https://github.com/nanu-c/axolotl/blob/181079e5e0fd7eb8832a8cf10a70f742fbf33bdd/axolotl-web/src/pages/Settings.vue#L48-L50

**Comment by Blackoverflow** [comment]
Hi,

I'm Blackoverflow, a UT user and software developer. I started helping by translating Signal Java code to golang, but that's (hopefully) not needed anymore, since we have crayfish now. Now, I want to focus on fixing small bugs and promote automated testing.

I'm not a very consistent contributor, but I hope I can add something, sometimes.

Ask me for help. :D

**Comment by kmsgli** [comment]
Notifications work fine with the app open it does not make noise but the notification light works and the lock screen shows you. I am talking about polling every 15min in case the app is closed ( I tend to close apps its just habit). So running axolotl from terminal on a systemd timer every 15 min works but I wonder if i can just run a texsecure command to check for new messages instead?
